bin/cache.ml

```diff
@@ -52,17 +52,42 @@ let trim =
          (User_message.make
             [ Pp.textf "Freed %s" (Bytes_unit.pp trimmed_bytes) ])
 
+let size =
+  let info =
+    let doc = "Query the size of the Dune cache" in
+    let man =
+      [ `P
+          "Queries the size of the Dune cache and outputs a human-readable \
```

Comment:
As in the `cache trim` command, I think it's useful to use the imperative voice when describing the functionality in comments or documentation, e.g.
```suggestion
          "Query the size of the Dune cache and output a human-readable \
```
I wonder if @christinerose has any opinion on this.

Reply:
Alternatively, "This command queries..." also seems fine (but a bit more verbose and repetitive, especially when seen together with other a few commands).

(But again, I don't have a strong opinion here and would appreciate some advice from @christinerose!)

Reply:
I've taken your suggestion into account, thanks!